packages/rocket-uploads3-store-event-aws-infrastructure/src/lambdas/index.ts

```diff
@@ -0,0 +1,44 @@
+import { AWSError } from 'aws-sdk'
+import { PutItemOutput } from 'aws-sdk/clients/dynamodb'
+
+export const handler = async (event: any): Promise<any> => {
+
+  const AWS = require('aws-sdk')
+  const { v4: uuidv4 } = require('uuid');
+  const ddb = new AWS.DynamoDB.DocumentClient()
+
+  const bucketName = event.Records[0].s3.bucket.name
+  const objectKey = event.Records[0].s3.object.key
+  const fileSize = event.Records[0].s3.object.size
```

Comment:
Super nitpicky: If you know the event content, could you change the type?
`async (event: <known_type>)`

Reply:
ah yes! I think the type is Records[], I will try :)